dubbo-rpc/dubbo-rpc-triple/src/main/java/org/apache/dubbo/rpc/protocol/tri/h3/Http3ClientFrameCodec.java

```diff
@@ -80,11 +110,38 @@ public void write(ChannelHandlerContext ctx, Object msg, ChannelPromise promise)
                 return;
             }
             ctx.write(new DefaultHttp3DataFrame(frame.content()), promise);
+        } else if (msg instanceof Http2PingFrame) {
+            sendPing((QuicChannel) ctx.channel());
```

Comment:
I think implementing a pingpong mechanism for http3 would result in clearer code than reusing http2.

Reply:
Considering using the control stream for heartbeat and graceful shutdown, but there is no documentation for this, so had to try it myself and used a simple solution for this version.

Reply:
Mark to be resolved later.